samples/test/transcoder.test.js

```diff
@@ -127,6 +127,24 @@ after(async () => {
   } catch (err) {
     console.log('Cannot delete bucket');
   }
+  // Delete outstanding jobs created more than 3 days ago
+  const {TranscoderServiceClient} =
```

Comment:
Is there a prefix we can check for as well on `job.name`? Otherwise I'm worried we might delete resources unintentionally.

Reply:
Unfortunately not. The job is a long-running operation where the server assigns a random name.